src/screens/Login.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  Image,
+  Pressable,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import React, { useState } from 'react';
+import AppButton from '../components/AppButton';
+import CheckBox from '@react-native-community/checkbox';
+
+const Login = () => {
+  const [secureTextEntry, setSecureTextEntry] = useState(true);
+  const [focusEmail, setFocusEmail] = useState(false);
+  const [focusPassword, setFocusPassword] = useState(false);
```

Comment:
🔧 Para variáveis que representam um valor booleano é interessante tentar deixar isso explícito através da nomenclatura da variável. Um exemplo seria mudar para:
```suggestion
  const [isSecureTextEntry, setSecureTextEntry] = useState(true);
  const [isEmailFocused, setEmailFocused] = useState(false);
  const [isPasswordFocused, setPasswordFocused] = useState(false);
```